config/default.schema.json

```diff
@@ -383,6 +390,7 @@
 		"BLOCK_DISPOSABLE_EMAIL_DOMAINS",
 		"HOST",
 		"ACTIVATION_LINK_PERIOD_OF_VALIDITY_SECONDS",
-		"LERNSTORE_MODE"
+		"LERNSTORE_MODE",
+		"PROMETHEUS"
```

Comment:
If i see right default values are set, why we need required at this position?

Reply:
objects must be required as they will not be build from defaults, and because of this defined in default.json